package.json

```diff
@@ -80,6 +80,7 @@
     "download": "^5.0.2",
     "filesize": "^3.3.0",
     "fs-extra": "^0.26.7",
+    "get-stdin": "^5.0.1",
```

Comment:
Is `get-stdin` worth to introduce rather than using Node.js built in `readline`?